Framework/Properties/Mantid.properties.template

```diff
@@ -186,6 +186,12 @@ loading.multifilelimit =
 # Hide algorithms that use a Property Manager by default.
 algorithms.categories.hidden=Workflow\\Inelastic\\UsesPropertyManager;Workflow\\SANS\\UsesPropertyManager;DataHandling\\LiveData\\Support;Deprecated;Utility\\Development;Remote
 
+# Response upon invoking an algorithm with its alias, when the alias is deprecated.
+# Allowed values are:
+#   "Warn": log a warning indicating the alias is deprecated, along with the name to be used
+#   "Raise": raise a RuntimeError if the deprecated deadline has been met
+algorithms.alias.deprecated = Warn
```

Comment:
Please be more linux-y and change these to be all lowercase (i.e. `warn` and `raise`)

Reply:
I'm following the convention of [PropertiesFile](https://github.com/mantidproject/mantid/blob/master/docs/source/concepts/PropertiesFile.rst). Looks like most options have their initials capitalized

Reply:
Reminding us that the first mantid developers were on windows :(